Support/Testing/CircleCI/Dockerfile

```diff
@@ -40,11 +40,11 @@ RUN apt-get install -y ninja-build
 RUN apt-get install -y flex bison
 
 # Install x86 compilers
-RUN apt-get install -y g++-4.9-multilib
+RUN apt-get install -y g++-multilib
 RUN apt-get install -y clang-5.0
 
 # Install arm compilers
-RUN apt-get install -y g++-4.9-multilib-arm-linux-gnueabi
```

Comment:
I don't know if you should touch these. The problem is just with multilib x86 vs. x86_64.

Reply:
Surprisingly these are not the ones that break. :D

Reply:
I think moving this is okay too, especially since it's practically free (in the sense that it's changing a few characters). It doesn't seem to break builds or tests (Android-ARM is actually running tests as I write this, but it's been flakey anyway). Do you feel strongly about keeping this at 4.9?

Reply:
No, I think you're right. I thought it might have been a mistake initially, but moving it makes sense.